main.py

```diff
@@ -274,7 +285,7 @@ async def get_local_fileformat(self):
         return self._fileformat
 
     async def loadConfig(self):
-        logger.info("Loading settings from: {}".format(os.path.join(settingsDir, "settings.json")))
+        logger.info("Loading settings from: {}".format(os.path.join(settingsDir, "decky-loader-settings.json")))
```

Comment:
I don't believe this is necessary since the settings are already inside a decky-recorder folder.

Reply:
Being clear, the settings file is already `decky-loader-settings.json`. All this is doing is making the console log match what the actual file is.